src/main/java/cart/domain/cart/Cart.java

```diff
@@ -0,0 +1,30 @@
+package cart.domain.cart;
+
+import cart.domain.member.Member;
+import cart.domain.product.Price;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cart {
+
+    private final List<CartItem> cartItems;
+
+    public Cart(final List<CartItem> cartItems) {
+        this.cartItems = new ArrayList<>(cartItems);
```

Comment:
```suggestion
        this.cartItems = cartItems;
```
카트아이템을 ArrayList로 굳이 명시하신 이유가 있나요??

Reply:
방어적 복사를 적용하고자 하였습니다!